Sources/BrowserServicesKit/PrivacyConfig/AppPrivacyConfiguration.swift

```diff
@@ -88,14 +96,65 @@ public struct AppPrivacyConfiguration: PrivacyConfiguration {
         default: return false
         }
     }
+    
+    private func isRolloutEnabled(subfeature: any PrivacySubfeature,
+                                rollouts: [PrivacyConfigurationData.PrivacyFeature.Feature.Rollout],
+                                randomizer: (Range<Double>) -> Double) -> Bool {
+        // Empty rollouts should be default enabled
+        guard !rollouts.isEmpty else { return true }
+        
+        let defsPrefix = "config.\(subfeature.parent.rawValue).\(subfeature.rawValue)"
+        if userDefaults.bool(forKey: "\(defsPrefix).\(Constants.enabledKey)") {
+            return true
+        }
+        
+        var willEnable = false
+        let rollouts = rollouts.sorted(by: { $0.percent < $1.percent })
```

Comment:
Maybe just in case, we should be removing duplicates and filtering out values outside of 0...100 range?